.gitignore

```diff
@@ -1,8 +1,8 @@
 /node_modules
 yarn.lock
 .json*
-package-lock.json
-package.json
+# package-lock.json
+# package.json
```

Comment:
These lines should just be deleted instead of commented out. In general leaving commented out code "just in case" leads to some messy looking files, and it should be the job the version control to store old code for later. If you need it back, you can just go into the git history to get it back.